source/databricks/calculation_engine/tests/features/given_a_wholesale_calculation/daily_tariff_per_ga_co_es/when_with_child_metering_point/test_output.py

```diff
@@ -29,4 +29,4 @@ def test__equals_expected(
     actual_and_expected: Any,
     output_name: str,
 ) -> None:
-    assert_output(actual_and_expected, output_name)
+    assert_output(actual_and_expected, output_name, True)
```

Comment:
What does "True" do?

Reply:
Parameter that only shows failing columns (similar to the way we only show mismatching rows)

Reply:
It's always hard to guess the semantics of boolean flags.
So this would be a nice improvement:
```suggestion
    assert_output(actual_and_expected, output_name, show_only_mismatching_rows=True)
```

Reply:
I don't disagree, but it's one for Alex/the framework, I guess. For what it's worth, the name of the parameter does show up when you view the file in Pycharm

Reply:
PyCharm is one thing. Reviewing and reading code is (often) another thing.
@energinet-ajw 👀